### ISSUE TITLE
Update Example Doc

### ISSUE THREAD

**Issue by abdinal1** [phuang1024/Starst3r#2]
In the docs you can find following example:

# Iterate point clouds with respect to each camera.
num_cameras = len(scene.pts3d)
for i in range(num_cameras):
    # Point cloud: pts shape is (N, 3); XYZ of each point.
    pts = scene.pts3d[i]
    print(f"Points from camera {i}: {pts.shape}")

    # Point colors: shape is (N, 3); RGB of each point.
    colors = scene.pts3d_colors[i]
    print(f"Colors from camera {i}: {colors.shape}")

# Iterate all points of all cameras.
for pt, color in starster.iterate_verts(scene):
    ...

num_verts = starster.num_verts(scene)
print("Total verts from all cameras:", num_verts)

But installing starst3r from pip, does not provide these functionalities e.g. 

scene.pts3d does not exisit, there is a num_cams variable and starster.iterate_verts does also not exisit.

Can you update the code or am i doing something wrong?

**Comment by phuang1024** [comment]
Sorry for the confusion. The library has been through a few refactors (including one right now).

For now, please follow the PointCloud docs: https://starst3r.readthedocs.io/en/latest/api.html#starster.PointCloud

Note that `starster.reconstruct_scene` returns a `PointCloud` instance.

For example:

```py
pts, colors = scene.pts_dense_flat()

for p in pts:
    ...
```

With the current refactor effort which will be finished soon, the API and docs will be cleaned up and updated.

**Comment by phuang1024** [comment]
Fixed with #3.
Docs are updated.
